RecoHGCal/TICL/plugins/SeedingRegionByTracks.h

```diff
@@ -0,0 +1,54 @@
+// Author: Felice Pantaleo,Marco Rovere - felice.pantaleo@cern.ch, marco.rovere@cern.ch
+// Date: 09/2018
+
+#ifndef __RecoHGCal_TICL_PRbyCA_H__
+#define __RecoHGCal_TICL_PRbyCA_H__
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h

Alternatively, this file can be removed in line with https://github.com/cms-sw/cmssw/pull/27439/files#r301672902

Reply:
ok  included guard matching the full file name